packages/@tailwindcss-cli/src/commands/build/index.ts

```diff
@@ -226,32 +233,36 @@ export async function handle(args: Result<ReturnType<typeof options>>) {
             fullRebuildPaths = inputFilePath ? [inputFilePath] : []
 
             // Create a new compiler, given the new `input`
-            ;[compiler, scanner] = await createCompiler(input)
+            ;[compiler, scanner] = await createCompiler(input, I)
 
             // Scan the directory for candidates
-            env.DEBUG && console.time('[@tailwindcss/cli] Scan for candidates')
+            DEBUG && I.start('Scan for candidates')
```

Comment:
These are called _after_ the initial `I.start('[@tailwincss/cli]')`, this allows us to remove the prefix because these logs will be indented.